designs/credentialManagement/ssoSupport.md

```diff
@@ -0,0 +1,32 @@
+# AWS SSO Support
+
+## Abstract
+
+This document describes the process of supporting Amazon SSO defined through a shared credentials profile.
+
+## Specification
+
+The [AWS SSO-OIDC service][SsoOidc] is SSO's implementation of the Device Authorization Grant flow as defined in [RFC8628][RFC8628].
+
+## SSO Login Sequence
+![][SsoLoginFlow]
+
+## Cache
+The SSO login flow contains two long lived tokens (`client registration id` and `access token`). These tokens MUST be cached in order to prevent the user from 
+having to re-do the SSO login flow whenever their assumed role session credentials expire. We currently only offer one implementation of the cache.
+
+### Disk Cache
+The disk cache is is written to allow interop of the Toolkit with other tools (e.g. AWS CLI) meaning that if they perform an SSO login in the terminal before
+starting the IDE, they do not need to perform it again and vice versa. The long lived tokens are cached in the `~/.aws/sso/` folder with `0600` permissions.
+
+## Profile keys
+SSO support adds the following keys to the AWS shared credential files:
```

Comment:
IMO it should read more like "The AWS shared credential file has these keys to support SSO", otherwise it reads like these are new things we are adding

Reply:
Tweaked